app/uk/gov/hmrc/gform/submission/SubmissionService.scala

```diff
@@ -80,6 +76,25 @@ class SubmissionService(
       )
     )
 
+  private def submission(
```

Comment:
I think you didn't do right thing in conflict resolution here.Yes yesterday we get rid of `private def submission` since it is not needed any longer:  386080f1

Reply:
done